### ISSUE TITLE
PointLinks_Test_Types_Of_PointLink_EventUpdate_And_EventChange

### ISSUE THREAD

**Issue by mateuszhyski** [SCADA-LTS/Scada-LTS#1148]

**Comment by mateuszhyski** [comment]
After test from my side, any of types Point Links ( it means Change or Update) do not work. Point Link work only, when point link have defines JavaScript.

**Comment by mateuszhyski** [comment]
Hi. The logic - solution in Scada which is called Point LInks , they works on mine branche Point_Links.

I and Michal, we tested the Poin Links in another way also, because I did not know that we have something like set logging type in point in data source.


So, finally they work but we saw that Virtual Data Source updates point value, propably additionally, because we saw twice rows in History Annotation in Point Details....one from Point Link and second one from another source (this source do not implement SetSourceType in Scada) - attachment shows it.

I will create a new branche , where I will put ONLY files which will have solution, because that one branche has a lot of files, which I did not touched. So to safety put same solution ( point link solution), I will create branche which will based on actual sivdevelop. I informed about this on today our scrum.
